src/config.py

```diff
@@ -52,8 +52,8 @@ class ZooKeeperConfig:
     def __init__(self, charm):
         self.charm = charm
         self.container = self.charm.unit.get_container(CONTAINER)
-        self.default_config_path = f"{self.charm.config['data-dir']}/config"
-        self.properties_filepath = f"{self.default_config_path}/zookeeper.properties"
+        self.default_config_path = f"{self.charm.config['data-dir']}/conf"
```

Comment:
Im not certain how this is set up in the ROCK, and if I said the wrong thing before I apologise, but I think we want the default config path, expected by the ROCK and the charm (if not specified) to be `/conf`
DataDir to be `/data` where myid and other stuff goes
Datalogdir to be `/datalog`

Reply:
I've made changes to the ROCK to support this and will be updating the PR shortly